packages/server/email/MailManagerDebug.ts

```diff
@@ -9,9 +9,12 @@ export default class MailManagerDebug extends MailManager {
     Subject: ${subject}
     Body: ${body}`)
 
+    let toStr = to
+    if (Array.isArray(to)) toStr = to.join(',')
```

Comment:
+1 nit:  we can avoid `let` here, or if you prefer avoid the variable all together by using the method call down below

```ts
const toStr = to.toString()
```